### DISCUSSION TITLE
Questions about the beg

### DISCUSSION THREAD

**Discussion by Kevin-Whoo** [artisticat1/obsidian-latex-suite#361] [Q&A] [answered]
```javascript
// Math mode
	{trigger: "mk", replacement: "$$0$", options: "tA"},
	{trigger: "dm", replacement: "$$\n$0\n$$", options: "tAw"},
	{trigger: "beg", replacement: "\\begin{$0}\n$1\n\\end{$0}", options: "mA"},
```
As shown above, the `mk` and `dm` work, but when i type `beg`, nothing happens, what should I do?

**Comment by ht-timchen** [comment]
It only work in the display math enviornment, e.g., between $$.

**Reply by Kevin-Whoo** [reply]
Thanks so much!

**Comment by anupchavan** [comment, marked as answer]
you have "mA" in the options, so it would work only in the math mode. If you want them to work everywhere, add "t" in the options: "tmA"

https://github.com/artisticat1/obsidian-latex-suite?tab=readme-ov-file#options

**Reply by Kevin-Whoo** [reply]
Thanks so much!
